plugins/module_utils/mh/base.py

```diff
@@ -63,3 +63,10 @@ def run(self):
         if 'failed' not in output:
             output['failed'] = False
         self.module.exit_json(changed=self.has_changed(), **output)
+
+    @classmethod
+    def execute(cls, module=None):
+        instance = cls(module)
+        instance.run()
+
+    make_it_so = execute
```

Comment:
Is this alias really necessary? Zen of Python suggests that there should be one -- and only one -- way to do something. Having multiple aliases introduces potential maintenance burden. Also, it's easier to add things than to remove them.

Reply:
I think it is necessary to have fun, but if that is too much, we can go along without.

Reply:
A framework I work on needs a method name. I have like 20 variants of how to name it and can't choose. I'm afraid that having multiple option names will impose the same cognitive load on the end-users here too. One could easily inherit the base class and have as many aliases as one wants. I don't think that this base class needs to introduce such a problem.

Reply:
OK, far enough, base classes are off limits for Capt. Piccard.